src/engine/SCons/Tool/mingw.py

```diff
@@ -150,6 +150,7 @@ def generate(env):
 
     #... but a few things differ:
     env['CC'] = 'gcc'
+    env['CCFLAGS'] = SCons.Util.CLVar('')
```

Comment:
with something like

`env=Environment(tools=['mslink','mingw','g++'],CCFLAGS='-myflag') `

with the new code CCFLAGS will be ignored?